### PR TITLE
OZGovernor Subgraphs Batch 2

### PR DESCRIPTION
**Pull request by danielkhoo**
## Description

Building on the OpenZeppelin Governor template #585. 

Adds the following new subgraphs:
- Unlock Protocol
- Angle Finance
- Code4Rena (Polygon)

## Misc
- Updated the deployment naming convention from `mainnet` to `ethereum` 
- Updated the main README to reflect the new OZGovernor subgraphs

### CODE REVIEW

File: subgraphs/openzeppelin-governor/protocols/angle-governance/src/governor.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+// Helper function that imports and binds the contract
+function getGovernanceFramework(contractAddress: string): GovernanceFramework {
+  let governanceFramework = GovernanceFramework.load(contractAddress);
+
+  if (!governanceFramework) {
+    governanceFramework = new GovernanceFramework(contractAddress);
+    let contract = Governor.bind(Address.fromString(contractAddress));
+
+    governanceFramework.name = contract.name();
+    governanceFramework.type = "OZGovernor";
```
Comment (mmnavarr) [inline subgraphs/openzeppelin-governor/protocols/angle-governance/src/governor.ts:110]: We should pull this out to a `const` now that its being re-used across the subgraphs. Will likely be adding more of these framework types which can go in the same place

File: subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        - Vote
+        - TokenHolder
+        - Delegate
+      abis:
+        - name: ArenaGovernor
+          file: ./abis/code4rena-governance/ArenaGovernor.json
+      eventHandlers:
+        # Part of GovernorPreventLateQuorum 
+        # Our entities currently don't track deadline
+        # - event: LateQuorumVoteExtensionSet(uint64,uint64)
```
Comment (mmnavarr) [inline subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml:29]: I'd be curious to know what fields are involved in this event and how they would effect the proposal entity fields

Feels like we should be handling it.. In a universe where we have a robust notification system (soon) this feels like a case that would be of importance to us

Comment (mmnavarr) [reply subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml:29]: Goes for the below `ProposalExtended` event as well

**Review by mmnavarr — DISMISSED** [review]
Minor feedback on `governanceFramework.type` options being moved to a const and some thoughts on the quorum extension

Would like to see the framework types migrated to const prior to merge but we should talk about quorum extension use case as a separate discussion (not blocking)

File: subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml (hunk 2 above)
Comment (danielkhoo) [reply subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml:29]: `LateQuorumVoteExtensionSet` changes the default number of blocks for an extension. imo it doesnt really serve any purpose to track.

But on second looks, i realise we can support `ProposalExtended` simply be updating the `endBlock`

Comment (danielkhoo) [reply subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml:28]: added tiny commit with extra handler for ProposalExtended

**Pull request merged**